spreadsheet/multiple-charts-from-csv/main.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/unidoc/unioffice/common/license"
+	"github.com/unidoc/unioffice/spreadsheet"
+)
+
+func init() {
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("Successfully set the Unidoc License Key")
+}
+
+type SalesData struct {
+	DateOfSale      time.Time
+	QuantitySold    float64
+	SalePrice       float64
+	SalespersonName string
+}
+
+func main() {
+	filePath := "./data/test-data.csv"
+	data, header, err := load_csv(filePath)
+	if err != nil {
+		panic("failed to load data ")
+	}
+
+	ss := spreadsheet.New()
+	defer ss.Close()
+	sheet := ss.AddSheet()
+
+	// Add header row
+	row := sheet.AddRow()
+	for _, h := range header {
+		row.AddCell().SetString(h)
+	}
+	row.AddCell().SetString("Cumulative Sum By Time")
+	row.AddCell().SetString("Cumulative Sum By Person")
+	// Add data rows.
+	cumulativeSumsByPerson := make(map[string]float64)
+
+	totalSum := 0.0
+	for _, dataRow := range data {
+		row := sheet.AddRow()
+		cell := row.AddCell()
+		cell.SetDate(dataRow.DateOfSale)
+		cellStyle := ss.StyleSheet.AddCellStyle()
+		cellStyle.SetNumberFormatStandard(spreadsheet.StandardFormat15) // format in d-mmm-yy
+		cell.SetStyle(cellStyle)
+
+		row.AddCell().SetNumber(dataRow.QuantitySold)
+		row.AddCell().SetNumber(dataRow.SalePrice)
+		row.AddCell().SetString(dataRow.SalespersonName)
+		totalSum += dataRow.SalePrice
+		row.AddCell().SetNumber(totalSum)
+
+		cumulativeSumsByPerson[dataRow.SalespersonName] += dataRow.SalePrice
+		row.AddCell().SetNumber(cumulativeSumsByPerson[dataRow.SalespersonName])
+	}
+
+	drawing := ss.AddDrawing()
+
+	// Create Line Chart showing daily sales.
+	chart, anc := drawing.AddChart(spreadsheet.AnchorTypeTwoCell)
+	// Set width to 8 cells
+	anc.SetWidthCells(10)
```

Comment:
in comment 8 cells, in code 10, why?

Reply:
I also have updated this comment, I am not sure why it is not updating here.